ui/src/app/shared/models.ts

```diff
@@ -721,3 +721,57 @@ export interface GnuPGPublicKey {
 }
 
 export interface GnuPGPublicKeyList extends ItemsList<GnuPGPublicKey> {}
+
+// https://kubernetes.io/docs/reference/kubectl/overview/#resource-types
+
+export const ResourceKinds = [
```

Comment:
ResourcesKinds should include capitalized singular kinds like `Pod`, `Deployment` etc.